docs/topics/mta-web-creating-custom-migration-targets.adoc

```diff
@@ -8,7 +8,7 @@
 
 Architects or users with `admin` permissions can create and maintain custom rulesets associated with custom migration targets. Architects can upload custom rule files and assign them to various custom migration targets. The custom migration targets are then available for selecting in the analysis configuration wizard.
 
-Using ready-made custom migration targets makes unnecessary the cumbersome task of configuring custom rules for each analysis run. This simplifies analysis configuration and execution for non-admin users or third-party developers and makes using the {ProductShortName} tool easier and more straightforward.
+Using ready-made custom migration targets makes unnecessary the cumbersome task of configuring custom rules for each analysis run. This simplifies analysis configuration and execution for non-admin users or third-party developers and makes using {ProductShortName} easier and more straightforward.
```

Comment:
nit: makes the cumbersome task of configuring custom rules for each analysis run unnecessary. 

IDK just sounds more correct to me, but I am no expert.